features/general/file-manager.md

```diff
@@ -0,0 +1,105 @@
+Especially when the file is **not associated** to a **specific document or record** in the ERP system, buth is a rather more **generic document** for business use, such as documentation regarding training courses, internal or external regulations, company practices, handbooks, and so on.  
+
+Users tend to keep such files in an external storage, like local drives or cloud platforms. But this often leaves a gap between navigating through internal ERP entities and external file storage.  
+
+A **File Manager inside the ERP system** is a **feasible and productive way** to **close this gap** and **utilize the ERP system's full potential**.  
+
+Your Business Management solution **@@name** comes with the updated functionality **File Management v2.0**.  
+
+**File Manager of @@name** enables you to **store and access** your files and folders in an organized and straightforward way without having to attach them to specific records. 
+Its intuitive structure resembles the well-known **Windows File Explorer structure** users are used to.  
+
+The new functionality for organizing your work files and folders is accessible to all users of your @@name platform, regardless of their workplace's physical location.  
+## File Manager Structure
+
+**File Management v2.0** unlocks new opportunities by providing **a uniform, centralized and shared structure** for managing your business documentation in a cloud environment, integrated with the rest of your data within the ERP platform. 
+
+**@@name's File Manager** uses a vertically-expandable **hierarchical tree structure** to arrange and represent your folders.  
+
+The **File tree** can be **expanded or collapsed** to **list or hide** subfolders for a more **detailed**, or conversely, a more **compact** layout.  
+You can use the **folder tree** to easily **view your folders** and **navigate throughout your directories structure**.  
+
+In the Folder tree you can also: 
+
+* Edit the definition (name, parent, and notes) of the selected folder.
+* Create a new folder
+* Change the name of the selected folder
+* Remove the selected folder from the folder tree and your storage
+* Refresh the contents of the folder tree  
+
+The **Files** window shows the files stored in the selected folder. It also allows to: 
+
+* Open the selected file using an appropriate application
+* Save the selected file to your local storage or external storage
+* Delete the selected file(s)
+* Upload new file(s) to your @@name File Manager storage
+
+## File Manager Views 
+
+The layout options for visualizing your files are identical to the popular ones in Windows File Explorer. 
+**Different layouts** represent a **different level of density and details** for the files in the current folder. 
+There are currently **eight available views** for viewing your documents: 
+
+* **Extra large icons** - displays the largest possible icons for files. Suitable for previews of images and thumbnails of video files 
```

Comment:
Describing all the views in a marketing material might be an overkill. You can just list them on the same line, like:
You can choose between views for different icon sizes, details, tiles, etc.

Reply:
The part with layouts was substantially reduced to make the article easier to perceive for newly-interested people.